### ISSUE TITLE
UserStory - SellBack Estimate

### ISSUE THREAD

**Issue by Aarondorn2** [binarygod/AmSYS#9]
As a ROLE_ADMIN, I would like a feature where I can paste a collection of items and view a 'sell back' price, so that I can sell items at reduced costs to alliance members or allies at a reduced price while maintaining a profit.

eg. add a setting for 'alliance sell back percentage' and 'allied sell back percentage' - a screen offers the same functionality as the buyback estimator, but with these percentage values instead.  Items can then be bought by the Corp at 5% tax and then sold to others at a 3% or 4% 'tax'.  This allows discounted prices for allies and alliance mates while still leaving a 1-2% margin for the Corp.

**Comment by binarygod** [comment]
A logistical question for you on this, is the corp keeping track of what is bought through the system (like everything gets dumped in to a cargo container), that way there's some knowledge of 'hey we don't have that much', and if you don't have that much will the corp acquire it for the player.  Will need to make sure that the aquired goods are at price corp paid for them....maybe if the need can't be met it gets shoved to the logic that will handle the 'lil buy this for me' system?

**Comment by Aarondorn2** [comment]
Yeah. I think that's the end goal. not sure if it can be released in smaller chunks or not.
Currently all 'stuff' is put in one hanger - can get a corp API and query that hanger

User pastes stuff in, and it get split into two 'queues' or at least two separate parts of a contract:
1) corp has stuff and can fill from inventory - user pays Jita buy for this
2) corp doesn't have stuff and it has to be hauled - user pays amarr sell + 10% (or configured price)

So user will only ever paste in one area, but may receive two separate contracts from the corp for their stuff at different times.

Thoughts?

**Comment by binarygod** [comment]
I like it!

**Comment by tehraven** [comment]
This is great.

**Comment by Aarondorn2** [comment]
slight modification on planned design:
first phase will not include active search of stuff that's "on hand", this will have to come later.

so far completed:
-front end UI for sell estimate
-front end UI for sell settings
-back end for sell settings (market, cache, settings)
-various code improvements (settings local cache, code refactors/ OOP, removal of obsolete classes & files)

still to do: 
-redesign dashboard
-front & backend for sell rules
-front and backend for 'feature entitlements' (who gets to see what). includes:
 --selection in authorization page to set entitlements at that level
 --selection in users page to set entitlements at the user level
